### PR TITLE
Revert transactional large blob entries

### PR DESCRIPTION
**Pull request by yamadapc**
This reverts a set of changes that added transaction semantics to large blob entries.

This means files that are spread amongst multiple files may be corrupted. We're revert to avoid other bugs related to this diff, possibly related to concurrency races and so on that will remain on the codebase.

Namely, we had issues where "large blobs" were missing when code expects it to be present.

Most of the time when reading "AssetGraph" large blob entry and sometimes at runtime possibly related to that entry being cleaned-up.

### CODE REVIEW

File: packages/core/cache/test/LMDBCache.test.js [hunk 1]
```diff
@@ -1,94 +0,0 @@
-// @flow
-
-import {DEFAULT_FEATURE_FLAGS, setFeatureFlags} from '@parcel/feature-flags';
```
Comment (mattcompiles) [inline packages/core/cache/test/LMDBCache.test.js:3]: Are all these tests broken now? Seems like most of them will still pass?

File: packages/core/core/test/test-utils.js [hunk 2]
```diff
@@ -54,7 +53,9 @@ export const DEFAULT_OPTIONS: ParcelOptions = {
     sourceMaps: false,
   },
   featureFlags: {
-    ...DEFAULT_FEATURE_FLAGS,
+    exampleFeature: false,
```
Comment (mattcompiles) [inline packages/core/core/test/test-utils.js:56]: Should this be `DEFAULT_FEATURE_FLAGS`

File: packages/core/cache/src/FSCache.js [hunk 3]
```diff
@@ -141,12 +132,12 @@ export class FSCache implements Cache {
...
-                  i * this.#writeLimitChunk,
-                  (i + 1) * this.#writeLimitChunk,
+                  i * WRITE_LIMIT_CHUNK,
+                  (i + 1) * WRITE_LIMIT_CHUNK,
                 )
               : contents.subarray(
-                  i * this.#writeLimitChunk,
-                  (i + 1) * this.#writeLimitChunk,
+                  i * WRITE_LIMIT_CHUNK,
+                  (i + 1) * WRITE_LIMIT_CHUNK,
```
Comment (mattcompiles) [inline packages/core/cache/src/FSCache.js:140]: Did we want to revert the chunking here too seeing as that was the potential issue you were concerned about?

File: flow-libs/lmdb.js.flow [hunk 4]
```diff
@@ -1,669 +0,0 @@
-// @flow
```
Comment (mattcompiles) [inline flow-libs/lmdb.js.flow:1]: Feels like a shame to revert this?

**Review by mattcompiles — APPROVED** [review]
I'll approve as I know this is a revert, but it feels like we're reverting code we'd like to keep too.

**Comment by mattcompiles** [comment]
Didn't realise auto-merge was on 😬

**Comment by yamadapc** [comment]
It's simpler to just revert the commits.

**Pull request merged**